pkg/build/buildkit/errors.go

```diff
@@ -73,13 +73,15 @@ func GetErrorMessage(err error, tag string) error {
 			E:    fmt.Errorf("buildkit service is not available at the moment"),
 			Hint: "Please try again later.",
 		}
-	case isPullAccessDenied(err):
-		if imageTag == "" {
+	case isPullAccessDenied(err) || isNotFound(err):
+		if imageTag == "" && isPullAccessDenied(err) {
 			imageTag = extractImageTagFromPullAccessDeniedError(err)
+		} else if imageTag == "" && isNotFound(err) {
+			imageTag = extractImageTagFromNotFoundError(err)
 		}
 		err = oktetoErrors.UserError{
-			E:    fmt.Errorf("failed to pull image '%s'. The repository is not accessible or it does not exist", imageTag),
-			Hint: fmt.Sprintf("Please verify the name of the image '%s' to make sure it exists.", imageTag),
+			E:    fmt.Errorf("the image '%s' is not accessible or it does not exist", imageTag),
+			Hint: fmt.Sprintf("Please verify the name of the image '%s' to make sure it exists.\nWhen using private registries, Okteto Registry Credentials are correctly configured https://www.okteto.com/docs/admin/registry-credentials/", imageTag),
```

Comment:
In other places we are just using golang multiline strings with the character "`". Look for other samples in the code

Reply:
@codyjlandstrom Could you review the message, please? It sounds a bit odd to me. I would say something like: `When using private registries, make sure Okteto Registry Credentials are correctly configured https://www.okteto.com/docs/admin/registry-credentials/` (notice the "make sure" addition in the message)

Reply:
i tried with `` and tabs / spaces but also was not getting the exact point. i will take a look.

regarding the copy i think i eat it with my coffee because clearly there is something missing :D thanks

Reply:
Updated as in description, I added a third line with the link i saw it too long for output, let me know any other copy change!